htdocs/src/Application/UserBundle/Admin/UserAdmin.php

```diff
@@ -11,11 +11,16 @@
 use Application\UserBundle\Entity\EntityManager\UserManager;
 use Application\UserBundle\Entity\User;
 
+/**
```

Comment:
@aivus @WhiteEagle88 We need to decide, should we add phpDoc to Class? I think, if we use something special in this class we should add a phpDoc description about this class, other way should stay empty